articles/root-cause-analysis.md

```diff
@@ -0,0 +1,79 @@
+---
+authorName: Matthias Meid
+authorGithubUsername: matthias-meid
+issue: 98
+title: You Always Have Time for a Proper Root Cause Analysis
+---
+# {{page.title}}
+
+A production issue will pop up, even in solid software. [Don't panic.](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Don't_Panic)
+
+We're more than eager to resolve the issue quickly. We should be, yet we should never stop being careful and analytical. Otherwise we might deploy another embarassing bug or suppress the problem without solving it, making everything worse.
+
+Saving an hour of analysis is not worth it.
```

Comment:
Den Satz musste ich zweimal lesen. Vermute wegen der doppelten Verneinung. Vorschlag:

Saving an hour of analysis is not worth it. -> Spending an hour of analysis is worth it.